mycroft/util/log.py

```diff
@@ -84,7 +84,11 @@ def init(cls):
         # Check configs manually, the Mycroft configuration system can't be
         # used since it uses the LOG system and would cause horrible cyclic
```

Comment:
just gonna leave this here https://github.com/HelloChatterbox/HolmesV/commit/0696559fd17f31ffb40f8099c839d98a23fd7096

Reply:
Cherry-picked this to remove the duplication of code. Very nice indeed